README.md

```diff
@@ -177,86 +177,130 @@ Parameter          | Type   | Description
 
 test('Check the menu item role', async t => {
-    const menuItem = await getMenuItem('Main Menu > Edit > Undo');
+    const menuItem = await getMainMenuItem(['Main Menu', 'Edit', 'Undo']);
     
     await t.expect(menuItem.role).eql('undo');    
 });
 ```
 
- ### getMainMenu
+### getContextMenuItem
+
+Gets a snapshot of the specified menu item from the **last** opened context menu.
+
+```js
+async function getContextMenuItem (menuItemSelector)
+```
+
+Parameter          | Type   | Description
+------------------ | ------ | -----
+`menuItemSelector` | String | An array of menu item labels and/or objects with properties "label" and "index".
 
- Gets a snapshot of the application main menu.
+ For example, you can pass the following values in the `menuItemSelector` parameter.
+
+ * `['Go To', 'Declaration']`
+ * `['Go To', { label: 'Declaration' }]`
+
+ If there are several menu items with the same label on the same level, you can specify an one-based "index"
+ property:
+
+ * `['Go To', {label: 'My Function', index: 2}]`
+
+ This value corresponds to the second menu item with label `My Function` in the `Go To` submenu.
+
+ Check the properties available in the snapshot
+ [here](https://github.com/electron/electron/blob/master/docs/api/menu-item.md).
+
+**Example**
+
+```js
+import { getContextMenuItem } from 'testcafe-browser-provider-electron';
+
+fixture `Electron test`
+    .page('./index.html');
+
+test('Check the menu item role', async t => {
+    await t.rightClick('.el');
+    
+    const menuItem = await getContextMenuItem(['Go To', {label: 'My Function', index: 2}]);
+    
+    await t.expect(menuItem.visible).ok();    
+});
+```
+
+ ### getMainMenuItems
+
+ Gets an array of snapshots of the items of the application main menu. If some item has a submenu, it also will be represented as an array of snapshots.
 
  ```
- async function getMainMenu ()
+ async function getMainMenuItems ()
  ```
 
- You can check properties available in the snapshot
- [here](https://github.com/electron/electron/blob/master/docs/api/menu.md).
+ You can check properties available in snapshots
+ [here](https://github.com/electron/electron/blob/master/docs/api/menu-item.md).
 
 **Example**
 
 ```js
-import { getMainMenu } from 'testcafe-browser-provider-electron';
+import { getMainMenuItems } from 'testcafe-browser-provider-electron';
 
 fixture `Electron test`
     .page('./index.html');
 
 test('Menu should contains the proper list of items', async t => {
-    const menuItems = (await getMainMenu()).items.map(item => item.label);
+    const menuItems = (await getMainMenuItems()).map(item => item.label);
     
     await t.expect(menuItems).eql(['File', 'Edit', 'Help']);
 });
 ```
 
- ### getContextMenu
+ ### getContextMenuItems
 
- Gets a snapshot of the **last** opened context menu.
+ Gets an array of snapshots of the items of the **last** opened context menu. If some item has a submenu, it also will be represented as an array of snapshots.
```

Comment:
```
Gets an array of item snapshots from the **most recently** opened context menu. If an item has a submenu, it will also be represented as an array of snapshots.
```